examples/mxnet_imagenet_resnet50.py

```diff
@@ -73,16 +72,23 @@
                     each bottleneck to 0 (default: False)')
 parser.add_argument('--model', type=str, default='resnet50_v1',
                     help='type of model to use. see vision_model for options.')
+parser.add_argument('--mode', type=str, default='symbolic',
+                    help='mode in which to train the model. options are \
+                    symbolic, hybrid (default: symbolic)')
```

Comment:
maybe use the name gluon instead of hybrid?

Reply:
Done